src/material/chips/chip-row.ts

```diff
@@ -135,6 +147,17 @@ export class MatChipRow extends MatChip implements AfterViewInit {
     }
   }
 
+  /** Sets _alreadyFocused (ahead of click) when chip already has focus. */
+  _handleMouseDown(event: MouseEvent) {
+    this._alreadyFocused = this._hasFocus();
```

Comment:
Might be worth binding this outside the `NgZone`.

Reply:
Good point.